medical_pathology/tests/test_medical_pathology.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from psycopg2 import IntegrityError
+
+from openerp.tests.common import TransactionCase
+from openerp.exceptions import ValidationError
+
+
+class TestMedicalPathology(TransactionCase):
+
+    def setUp(self):
+        super(TestMedicalPathology, self).setUp()
+        self.pathology_1 = self.env.ref(
```

Comment:
The second test case doesn't use this attribute, so it's 1 of 2. In any event, it's a pretty minor improvement and doesn't really need to happen.